### ISSUE TITLE
Fix URLs point to TFL

### ISSUE THREAD

**Issue by blendsdk** [classic-terra/classic-docs-next#2]
There are several URL pointing to TFL that need to be fixed. These preferably should be pointing the the community docs domain

**Comment by LuncBurner** [comment]
Yes, this is correct.  Let's point to the community docs domain.
